src/MPoly.jl

```diff
@@ -1655,5 +1655,7 @@ true
 Like [`polynomial_ring(R::Ring, s::Vector{Symbol})`](@ref) but return only the
 multivariate polynomial ring.
 """
-polynomial_ring_only(R::T, s::Vector{Symbol}; internal_ordering::Symbol=:lex, cached::Bool=true) where T<:Ring =
-   mpoly_ring_type(T)(R, s, internal_ordering, cached)
+function polynomial_ring_only(R::T, s::Vector{Symbol}; internal_ordering::Symbol=:lex, cached::Bool=true) where T<:Ring
```

Comment:
This of course conflicts with PR #1729 (no biggie, just want to make it explicit -- we'll have to plan in time to adjust depending on the order we merge these in)